lib/omni_auth/multi_provider.rb

```diff
@@ -13,6 +13,7 @@ def self.register(builder,
 
       handler = OmniAuth::MultiProvider::Handler.new(path_prefix: path_prefix,
                                                      identity_provider_id_regex: identity_provider_id_regex,
+                                                     **options,
```

Comment:
I think it's fine. Can you also remove the explicit declaration of the `identity_provider_id_regex` parameter from the `register` signature so it just gets passed through with the `options`?